extensions/wikia/Wall/tests/CommentsIndexTest.php

```diff
@@ -0,0 +1,110 @@
+<?php
+
+class CommentsIndexTest extends WikiaBaseTest {
+	/** @var int $parentId */
+	private $parentId;
+	/** @var int $commentId */
+	private $commentId;
+	/** @var CommentsIndexEntry $parentEntry */
+	private $parentEntry;
+	/** @var CommentsIndexEntry $entry */
+	private $entry;
+
+	public function setUp() {
+		$this->setupFile = __DIR__ . '/../Wall.setup.php';
+		parent::setUp();
+
+		$this->parentId = 1526;
+		$this->commentId = 1686;
+		$this->parentEntry = new CommentsIndexEntry();
+		$this->parentEntry->setCommentId( $this->parentId );
+		$this->entry = new CommentsIndexEntry();
+		$this->entry
+			->setCommentId( $this->commentId )
+			->setParentCommentId( $this->parentId );
+	}
+
+	/**
+	 * Verify that CommentsIndex populates its in memory cache with queried entry instances
+	 */
+	public function testCommentsIndexUsesInObjectCacheForEntries() {
+		$dbMock = $this->getDatabaseMock( [ 'selectRow' ] );
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Define a constant instead of duplicating this literal "selectRow" 4 times. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1192)